Services/Style/classes/class.ilStyleDefinition.php

```diff
@@ -1,4 +1,4 @@
-<?php
+	<?php
```

Comment:
Why indented?

Reply:
This was not intended and may be happened as I replaced the spaces with tabs. I will remove it and review the code for further wrong indentations.